mergify_engine/tests/unit/actions/test_merge.py

```diff
@@ -271,3 +272,23 @@ def test_queue_summary_subscription(active, summary):
     with mock.patch.object(merge.queue.Queue, "from_context", return_value=q):
         action = merge.MergeAction(voluptuous.Schema(merge.MergeAction.validator)({}))
         assert summary == action.get_queue_summary(ctxt, q)
+
+
+def test_strict_merge_parameter():
```

Comment:
That's good.
You could use `pytest.mark.parametrize` to run all the scenario there but no big deal 😄